android/src/main/kotlin/be/tramckrijte/workmanager/WorkmanagerCallHandler.kt

```diff
@@ -145,14 +145,16 @@ private object UnregisterTaskHandler : CallHandler<WorkManagerCall.CancelTask> {
         convertedCall: WorkManagerCall.CancelTask,
         result: MethodChannel.Result
     ) {
-        when (convertedCall) {
-            is WorkManagerCall.CancelTask.ByUniqueName -> WM.cancelByUniqueName(
-                context,
-                convertedCall.uniqueName
-            )
-            is WorkManagerCall.CancelTask.ByTag -> WM.cancelByTag(context, convertedCall.tag)
-            WorkManagerCall.CancelTask.All -> WM.cancelAll(context)
-        }
+        try {
+            when (convertedCall) {
+                is WorkManagerCall.CancelTask.ByUniqueName -> WM.cancelByUniqueName(
+                    context,
+                    convertedCall.uniqueName
+                )
+                is WorkManagerCall.CancelTask.ByTag -> WM.cancelByTag(context, convertedCall.tag)
+                WorkManagerCall.CancelTask.All -> WM.cancelAll(context)
+            }
+        } finally {}
```

Comment:
Please handle the errors appropriately - what could go wrong here?

Reply:
Probably, we don't need it, I will remove.